Battle.java

```diff
@@ -0,0 +1,102 @@
+import java.util.Random;
+
+public class Battle {
```

Comment:
A Java ilyen nyelv, hogy osztalyok nelkul nem nagyon letezhetnek fuggvenyek. De itt gyakorlatilag a konstruktorban beallitod a ket entity-t, aztan 1x lefuttathatod a csatat rajtuk. (Tobbszor nem, mert mar megurdelt az egyik ugye). Mas nyelveken erre az ember irna egy szimpla fuggvenyt. Java-ban ez nem opcio, de static fuggvenyeket tudsz csinalni, es itt szreintem indokoltabb is volna, ha minden fuggveny static volna, es a `battle` szimplan megkapna a ket Entity-t.